### ISSUE TITLE
Open PWA directly in browser language

### ISSUE THREAD

**Issue by ximfritz** [intershop/intershop-pwa#1490]
**Is your feature request related to a problem? If yes, please describe it.**

If the PWA has multiple languages the PWA should on the first visit open the PWA in the browser language if this language is available.

I am always frustrated when i use my english browser and the pwa open up with the language de. Although the PWA has two languages de and en.

**Describe the desired solution.**

The PWA redirects the website on the first load  to the browser specific language.

1- Start my browser who uses the english language
2 - Type in url
3 - Press enter
4 - The shop uses the english language

**Describe alternatives you've considered.**

**Provide additional context.**
If you have a shop with 10 languages where you have customers from all over the world it is annoying for each customer to switch the language on the first visit.

[AB#88822](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/88822)

**Comment by shauke** [comment]
We will discuss this, but there was already a pull request for this that was not merged since we did not actually wanted this functionality. See https://github.com/intershop/intershop-pwa/pull/583

**Comment by SGrueber** [comment]
Maybe PR #1447 would solve this problem

**Comment by ximfritz** [comment]
@SGrueber Do you mean would solve this feature, or which problem do you mean?

**Comment by SGrueber** [comment]
@ximfritz You wrote: _I am always frustrated when i use my english browser and the pwa open up with the language de. Although the PWA has two languages de and en._

And I guessed to save your language selection in a cookie would solve this problem.

**Comment by SGrueber** [comment]
@ximfritz Yes, there is a redirect to the correct language on pwa start - we will add a feature toggle for the 'save language as cookie' soon and afterwards we will merge this PR

**Comment by ximfritz** [comment]
@SGrueber I think we're talking past each other, I think i was not very clear.
 The user visit the shop for the first time and never accept any cookies or did a language switch. The browser language is de, the PWA application has multiple languages and normally starts with the language en. 
 
 But instead of starting with en it starts with de, because of the browser language.
